elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/ColumnContext.java

```diff
@@ -233,4 +229,9 @@ private Object resolveSQLHandlebar(final Object context, final Options options)
 
         return columnArgMap;
     }
+
+    public static Map<String, Argument> mergedArgumentMap(Map<String, Argument> referencedColumnArgs,
```

Comment:
javadoc

Reply:
These function parameters use terms like column but the function is also leveraged for table arguments.  Let's rename these parameters so they make sense in both contexts.